README.md

```diff
@@ -0,0 +1,12 @@
+# Vault Operator (Kubernetes)
+[![CharmHub Badge](https://charmhub.io/vault-k8s/badge.svg)](https://charmhub.io/vault-k8s)
+
+Vault is a tool for securely managing secrets used in modern computing (e.g. passwords, certificates, API keys).
+
+The Vault Operator deploys and initializes Vault on Kubernetes, and runs in high availability mode using the Raft backend.
+
+For more information including guides, integrations, configuration options, see [Vault's Charmhub page](https://charmhub.io/vault-k8s).
+
+## OCI Images
+
+- Vault: [ghcr.io/canonical/vault](https://github.com/canonical/vault-rock)
```

Comment:
I'm wondering if you are missing a [project and community](https://github.com/canonical/indico-operator?tab=readme-ov-file#project-and-community) section here

Reply:
Fixed in #499